dashboard/app/models/user_ml_model.rb

```diff
@@ -0,0 +1,14 @@
+# == Schema Information
+#
+# Table name: user_ml_models
+#
+#  id         :bigint           not null, primary key
```

Comment:
Most of our other models seem to use `:integer`?

Reply:
We've bumped up against the maximum SQL integer limit on a few tables, so starting out with bigint seems appropriate.